projector-html-runtime-purs/src/Projector/Html/Runtime/Pux.purs

```diff
@@ -0,0 +1,55 @@
+module Projector.Html.Runtime.Pux (
+    Html
+  , text
+  , textUnescaped
+  , parent
+  , void
+  , fold
+  , blank
+  , attr
+  ) where
+
+import Data.Array as Array
+import Data.Function.Uncurried (runFn3)
+import Prelude
+import Pux.Html.Attributes (attr) as Pux
+import Pux.Html.Elements (Html, Attribute, element, text) as Pux
```

Comment:
Not sure where we left it last, but is it possible just to change this to be a type-class drive implementation rather than pux specific? Pux would then just be one implementation, but we could switch to something else if desired.

Reply:
Yeah I'll have a go at that tomorrow, it's a small step now that this work is done.

Rather than a typeclass I'll probably do something simple, like assuming `Html` et al are in scope and letting you pass in a runtime import.